strings/src/main/res/values/strings.xml

```diff
@@ -525,9 +525,9 @@
     <string name="invalid_email_address">Invalid email address!</string>
     <string name="sort_by">Sort by</string>
     <string name="server">Server</string>
-<!--
-    <string name="submission_transport_admin_setting">Submission transport</string>
--->
+    <!--
```

Comment:
This was an oversight when removing the SMS feature. Would be great to remove it entirely at some point.

Reply:
I'll do it in https://github.com/getodk/collect/issues/4072